test/basic.test.ts

```diff
@@ -29,6 +29,16 @@ test('switch schema', async () => {
   expect(res).toMatchSnapshot()
 })
 
+test('on_conflict insert', async () => {
+  let res = await postgrest
+    .from('users')
+    .insert([
+      { username: 'izqalan', catchphrase: 'bat', status: 'OFFLINE', age_range: [20, 25]},
+      { username: 'dragarcia', catchphrase: "'fat' 'cat'", status: 'OFFLINE', age_range: [20, 30]}
+    ], { upsert: true , on_conflict: 'username'});
+  expect(res).toMatchSnapshot()
+})
+
```

Comment:
I think you can make a minimal test with something like:

```js
const res = await postgrest
  .from('users')
  .insert({ username: 'dragarcia' }, { upsert: true, onConflict: 'username' })
expect(res).toMatchSnapshot()
```

This preserves the state of the database, leaving other tests unaffected. In an ideal world, each test would be independent from each other, but this is the best we can come up with right now.

Reply:
ah, that's better